src/Uno.Extensions.Reactive.Tests/_Utils/SourceContextExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.Extensions.Reactive.Core;
+
+namespace Uno.Extensions.Reactive.Tests
+{
+	internal static class SourceContextExtensions
+	{
+		/// <summary>
+		/// DO NOT USE - FOR TESTS PURPOSES ONLY - Creates a child context given a request source.
+		/// </summary>
+		/// <param name="requests">The request source to use for the new context.</param>
+		/// <param name="owner">The name of the owner of this child context.</param>
+		/// <returns>A new child SourceContext.</returns>
+		internal static SourceContext CreateChild(this SourceContext ctx, IRequestSource requests, [CallerMemberName] string owner = "")
```

Comment:
Because it's an extension, maybe make it more scoped?

```suggestion
		internal static SourceContext CreateChildContext(this SourceContext ctx, IRequestSource requests, [CallerMemberName] string owner = "")
```

Reply:
Those are only overloads for the main method which is `CreateChild`. I would keep them like this. BTW it's only `internal` stuff, we can change it later if we prefer.